neuro_evolution_ctrnn/brains/CNN_CTRNN.py

```diff
@@ -99,7 +99,7 @@ def __init__(self, input_space: Space, output_space: Space, individual: np.ndarr
         self.eval()
 
     def brainstep(self, ob):
-        x: torch.Tensor = torch.tensor([ob], dtype=np.float32).permute(0, 3, 1, 2)
```

Comment:
Ist das alte nicht besser?

Reply:
hat mir fehler gegeben `TypeError: tensor(): argument 'dtype' must be torch.dtype, not type` und daher habe ich einfach quick&dirty den alten stand genommen. 

ich habe  jetzt mal genau hingeschaut. wenn ich `x: torch.Tensor = torch.tensor([ob], dtype=torch.float32).permute(0, 3, 1, 2)` hinschreibe, scheint es zu gehen

Aber dann habe ich mal ein paar experimente gemacht. mit `torch.tensor` dauert ein kleines experiment 30-35 sekunden. mit `torch.from_numpy` dauert das gleiche experiment 8-10 sekunden. 

Daher würde ich das jetzt erstmal so lassen. Wobei ich anmerken muss, dass ich keine Ahnung habe, was hier eigentlich passiert, und das ist alles nur trial&error